omp/__init__.py

```diff
@@ -7,10 +7,11 @@
 import sys
 
 try:
+    # there may be an environ modification when loading config
     from pythran import config
-    cxx = config.cfg.get('user', 'cxx')
 except ImportError:
-    cxx = 'c++'
+    pass
+cxx = os.environ.get('CXX', 'c++')
```

Comment:
Hum, OK I though that c++ was always valide.

I looked at the code using the cxx compiler and finally notice that it does ... nothing.
If the commande return useful information, it means the library is in the LIBRARY_PATH or default search (this is exactly what it done by find_library).
Shouldn't we just remove this part?